maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/ConsoleOutputFileReporter.java

```diff
@@ -69,8 +69,8 @@ public void close()
         {
             try
             {
-                fileOutputStream.flush();
-                fileOutputStream.close();
+                // fileOutputStream.flush(); Will not call close on exception!
+                fileOutputStream.close(); // Will implicitly flush.
```

Comment:
I just wanted to be sure that `close` is always called. See the comment. When `flush` throws an exception, `close` will never be called and leak resources.